### PR TITLE
[Frontend] add box with info about api identifiers

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | 1.8
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

image

### CODE REVIEW

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "swagger-info-box"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by AdamKasp — APPROVED** [review]

**Review by Tomanhez — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig [hunk 1]
```diff
@@ -61,6 +79,17 @@
     <div class="webby"><img src="{{ asset('bundles/apiplatform/webby.png') }}"></div>
 {% endif %}
 
+<div class="information-container wrapper" id="info-swagger">
+    <h2 class="title">
+        <small>
+            Parameter {id} used in API endpoints is in fact API resource identifier, configured in API resource configuration file.
+            Order API resource id is tokenValue.
+            Customer API resource id is id.
+            Rest of the endpoints resource id is code.
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig:88]:
```suggestion
            For the rest of the endpoints, the id of the resource is code.
```

File: src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig [hunk 2]
```diff
@@ -61,6 +79,17 @@
     <div class="webby"><img src="{{ asset('bundles/apiplatform/webby.png') }}"></div>
 {% endif %}
 
+<div class="information-container wrapper" id="info-swagger">
+    <h2 class="title">
+        <small>
+            Parameter {id} used in API endpoints is in fact API resource identifier, configured in API resource configuration file.
+            Order API resource id is tokenValue.
+            Customer API resource id is id.
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig:87]:
```suggestion
            For the Customer API, the id of the resource is id.
```

File: src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig [hunk 3]
```diff
@@ -61,6 +79,17 @@
     <div class="webby"><img src="{{ asset('bundles/apiplatform/webby.png') }}"></div>
 {% endif %}
 
+<div class="information-container wrapper" id="info-swagger">
+    <h2 class="title">
+        <small>
+            Parameter {id} used in API endpoints is in fact API resource identifier, configured in API resource configuration file.
+            Order API resource id is tokenValue.
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig:86]:
```suggestion
            For the Order API, the id of the resource is tokenValue.
```

File: src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig [hunk 4]
```diff
@@ -61,6 +79,17 @@
     <div class="webby"><img src="{{ asset('bundles/apiplatform/webby.png') }}"></div>
 {% endif %}
 
+<div class="information-container wrapper" id="info-swagger">
+    <h2 class="title">
+        <small>
+            Parameter {id} used in API endpoints is in fact API resource identifier, configured in API resource configuration file.
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Resources/views/SwaggerUi/index.html.twig:85]:
```suggestion
            Parameter {id} used in API endpoints is in fact API resource identifier, configured in the API resource configuration file. The following rules apply to this parameter:
```

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @SirDomin! :tada:

**Pull request merged**